autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -216,7 +196,7 @@ def test_create_nonzero_scaling_group_with_pool_on_cloud_network(self):
             asserter=self)
         self.verify_group_state(pool_group_resp.entity.id, self.min_servers)
 
-    @tags(speed='slow', type='rcv3', rcv3_mimic='fail')
+    @tags(speed='slow', type='rcv3', rcv3_mimic='fail', convergence='error')
```

Comment:
Oh good point, thanks.  :)  It's just that it's testing for desired capacity going down in response to an error.  I'll change these to 'never' instead, as well.